src/pages/Dashboard/DashboardTeam/SalesLogTable.js

```diff
@@ -0,0 +1,96 @@
+import React, { useMemo } from 'react';
+import { useTable, useSortBy } from 'react-table';
+
+import { TriangleDownIcon, TriangleUpIcon } from '@chakra-ui/icons';
+import { Box, Skeleton, Table, Thead, Tbody, Tr, Th, Td, chakra } from '@chakra-ui/react';
+
+const SalesLogTable = ({ loading, data }) => {
+  const sales = useMemo(
+    () =>
+      data?.getLatestOrders?.map(order => ({
+        date: new Date(parseInt(order.createdAt, 10)).toDateString(),
+        order: order.orderNumber,
+        qty: order.numberOfItems,
+        price: '$' + order.price.toFixed(2),
+      })) ?? [],
+    [data],
+  );
+
+  const columns = useMemo(
```

Comment:
Does this need to be wrapped in a memo?

Reply:
Interesting question! I did some research: https://react-table.tanstack.com/docs/quick-start

> It's important that we're using React.useMemo here to ensure that our data isn't recreated on every render. If we didn't use React.useMemo, the table would think it was receiving new data on every render and attempt to recalculate a lot of logic every single time. Not cool!

u can look into `useMemo` if you'dl like, it's kinda complicated